x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -12,6 +12,15 @@
 # query = 'process where serial_event_id = 1'
 # expected_event_ids  = [1]
 
+
+# fails because of string check - msbuild does not match MSBuild
```

Comment:
@astefan @rw-access case sensitivity biting back. We need to figure out where we stand on this by the end of the month.

Reply:
My other PRs for case sensitivity and the updates to the tests themselves should help